src/ml/neural_net/style_transfer/mps_style_transfer_decoding_node.mm

```diff
@@ -77,27 +77,27 @@ - (MPSNNImageNode *) backwardPass:(MPSNNImageNode *) inputNode {
 }
 
 - (void) setLearningRate:(float)lr {
-  [[_conv weights] setLearningRate:lr];
-  [[_instNorm weights] setLearningRate:lr];
+  [[_conv tcWeightsData] setLearningRate:lr];
```

Comment:
Nit: This statement, and others like it, would be more readable as e.g.
```
[_conv.tc_weights setLearningRate:lr];
```
May as well actually use the property syntax since we've defined our category methods as a property